dbtype/mssql/db_report_string.sh

```diff
@@ -7,9 +7,9 @@ db_report_string() {
   then
     ${db_binary} -d ${deployment_db} ${server_flag}${port_flag} ${user_flag} ${password_flag} -h -1 -b -Q "
     SET NOCOUNT ON;
-    SELECT '${db_basedir}/' + dbname + '/' + deployment_type + '/' + deployment_name
+    SELECT '${db_basedir}/' + ${dbname} + '/' + deployment_type + '/' + deployment_name
```

Comment:
I'm getting an error running an update on SQL Server.  I think this line is the problem.  I don't think we needed to make this change.  dbname was referring to the column here, not the variable(though they are equal).  Even so I think we would want db_destination_name here.  Probably a similar issue in PG.